unsubscribe.html

```diff
@@ -353,72 +360,100 @@ <h5 id="unsubscribe-intro">{% filter ak_text:"unsubscribe_intro" %}{% endfilter
 					<div id="id_email_box" class="input-text ak-input-type-user required ak-err-below">
 						<label for="id_email">{% filter ak_text:"field_email" %}Email Address{% endfilter %}</label>
 						<input type="text" name="email" id="id_email" class="ak-userfield-input">
-					</div>			 
-	<!--		   
+					</div>
+	<!--
 				<div class="ak-labels-overlaid">
 					<div>
 						<label for="id_email">{% filter ak_text:"field_email"|capfirst %}{% endfilter %}</label>
 						<input id="id_email" type="text" name="email" size=40 />
 					</div>
 					<input type="hidden" name="country" value="United States" />
 				</div>
-	-->	
+	-->
 			</div>
 			<div id="known_user">
-			<div>{% filter ak_text:"unsubscribe_user_warning_prename " %}Not{% endfilter %} <strong><span id="known_user_name"></span></strong>{% filter ak_text:"unsubscribe_user_warning_postname " %}?{% endfilter %} <a href="/unsubscribe/unsubscribe">{% filter ak_text:"unsubscribe_user_warning_link_label" %}Use this form instead{% endfilter %}</a></div>				
+				<div>
+					{% filter ak_text:"unsubscribe_user_warning_prename " %}Not{% endfilter %}
+					<strong>
+						<span id="known_user_name"></span>
+					</strong>
+					{% filter ak_text:"unsubscribe_user_warning_postname " %}?{% endfilter %}

```

Comment:
If trailing space in filter names is not intentional, should be removed.
Also, on [L359](https://github.com/350org/ak_intl_v3/blob/1919f44ff0353dbcd6ba3b2aae4292a996bf5f2f/unsubscribe.html#L359) should default text be added?